source/fundamentals/crud/read-operations/project.txt

```diff
@@ -57,149 +74,132 @@ method in the options of the following read operation methods:
 Exclude a Field
 ~~~~~~~~~~~~~~~
 
-To exclude a field, pass the field you want to exclude and a ``0`` to
-the ``SetProjection()`` method. For all fields you don't explicitly
-list in the projection, the driver includes them.
+To exclude a field, pass the field you want to exclude with a ``0`` to
+the ``SetProjection()`` method. The driver includes all fields that are
+not explicitly excluded in the projection document.
```

Comment:
Question:

Is this accurate? Just curious because it seems odd. If that's the case, why would you ever need to include a field? I might be missing something. This also doesn't match `$project`, but I am beginning to think that is expected.

Reply:
I think the purpose is for documents with many fields, it is faster to either include or exclude just a few fields instead of the counterpart. For example, if a document had 30 fields and you only wanted to include two fields in a projection, you could specify an include, instead of excluding 28 fields. For the example data I used in this page (only 3 fields), I can see why it seems unclear.